docs/src/main/sphinx/installation/cli.rst

```diff
@@ -540,6 +532,49 @@ and displays an error message (which is unaffected by the output format):
     Query 20200707_170726_00030_2iup9 failed: line 1:25: Column 'region' cannot be resolved
     SELECT nationkey, name, region FROM tpch.sf1.nation LIMIT 3
 
+.. _cli-output-format:
+
+Output formats
+--------------
+
+The Trino CLI provides the options ``--output-format``
```

Comment:
Why the weird wrapping?

Reply:
I meant on the whole paragraph .. somehow looks off to me .. anyway .. no worries .. I am just used to rewrap extension usage in vscode..